marcgarreau/accelerator
# Circle config

#### What's this PR do?

Provides a base CircleCi config that will enable junit xml test metadata, which is needed for the Heartbeat application

#### Where should the reviewer start?

your call

#### How should this be manually tested?

You could make these exact changes to your unejected create-react-app based app that is already registered correctly in Heartbeat and see if the build data flows into KPI

#### Any background context you want to provide?

no

#### New dependencies? What are they used for?

jest-junit to produce the junit xml

#### Screenshots (if appropriate)

#### What gif best describes how this PR makes you feel?

![addy-vroom](https://user-images.githubusercontent.com/3867836/29215174-2d06fdc8-7e67-11e7-85bf-8889a307d500.gif)
